src/status_im2/common/json_rpc/events.cljs

```diff
@@ -45,7 +45,7 @@
                            (transforms/js->clj (.-result response-js)))))))))))
 
 (re-frame/reg-fx
- ::call
+ :call
```

Comment:
:json-rpc/call

Reply:
@yqrashawn this is important, pls fix this

Reply:
oops

Reply:
fixed..